### PR TITLE
bug 1519622. Disable rollback of ES DCs

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @portante

**Comment by jcantrill** [comment]
@portante I only did so in the event someone squawks but am happy to remove the configurability if it makes sense otherwise;  this is the reason its 'private'; to make it more difficult to change

File: roles/openshift_logging_elasticsearch/vars/main.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ __latest_es_version: "3_6"
 __allowed_es_versions: ["3_5", "3_6", "3_7"]
 __allowed_es_types: ["data-master", "data-client", "master", "client"]
 __es_log_appenders: ['file', 'console']
+__es_revision_history_limit: 0
```
Comment (ewolinetz) [inline roles/openshift_logging_elasticsearch/vars/main.yml]: is this something we expect customers to set? if so this doesn't belong in `vars/main.yaml` but rather `defaults/main.yaml`

**Comment by ewolinetz** [comment]
bot, retest this please

**Comment by ewolinetz** [comment]
bot, retest this please

**Comment by ewolinetz** [comment]
bot, retest this please

**Pull request merged**